spec/OpenLineage.json

```diff
@@ -443,6 +446,41 @@
         }
       ]
     },
+    "ExpectationsAssertionsDatasetFacet": {
+      "description": "list of tests performed on dataset or dataset columns, and their results",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseFacet"
+        },
+        {
+          "type": "object",
+          "required": ["assertions"],
+          "properties": {
+            "assertions": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "expectationType": {
+                    "type": "string",
+                    "description": "Type of expectation test that dataset is subjected to",
+                    "example":  "not_null"
+                  },
```

Comment:
Where possible, I think it would be great if the assertion had the expected value

Reply:
Do you mean adding additional field that contains argument to given expectation? For example, for expectation `expect_table_row_count_to_equal` the value could be `7`. 

I think if we want to do that, the better approach would be to accept untyped dictionary with arguments - the argument count and type can wildly differ between expectations, for example: https://github.com/calogica/dbt-expectations#expect_column_values_to_be_within_n_moving_stdevs